docs/testplans/study-viewer/study-viewer.md

```diff
@@ -24,12 +24,21 @@ It enables to the user to check :
 ## Tests Scenarios
 
 1. User with no access - <br>
-a. go to page, click on `Request Access` button to request access, recives `request_id` from the Request Access Queue <br>
-b. makes a manual call to validate the `request_id` with the Requestor <br>
-c. go back to the back, click on `Download` button to download the dataset <br>
-
-2. User with access - go to page, click on `Download` button and download the file from indexd
+1.1. User does not login and requests the access <br>
+  a. go to study page, sees `Login to Request Access` button, clicks the button <br>
+  b. redirected to the `Login Page`. The user logs in with valid credentials  <br>
+  c. goes back to the study page, sees `Request Access` button to request access for the dataset <br>
+1.2. User logs in and request access <br>
+  a. logs in and goes to the study page, clicks on `Request Access` button <br>
+  b. the user receives `request_id` <br>
+  c. using `request_id` received, request to requestor service is send to update the status of the request to `APPROVED` <br>
+  d. user refreshes the study page and sees button disabled with message `DAR In Progress`
+  e. with the same `request_id`, another request to requestor service is send to update the status of the request to `SIGNED` <br>
+  f. user refreshes the study page and sees `Download` button displayed where the user can now download the study
+
+2. User with access - user logs in and goes to study page, click on `Download` button and download the file from indexd
  
 3. `Learn More` button navigates the user to `one row` page which displays the requested dataset
 
-4. Test functionality in a Data Commons with multiple configured Study Viewers
```

Comment:
you shouldn't remove `Test functionality in a Data Commons with multiple configured Study Viewers`. It was not implemented but it's still a test case that should be tested.